packages/apple-pay-integration/src/apple-pay-button-strategy.ts

```diff
@@ -313,9 +403,17 @@ export default class ApplePayButtonStrategy implements CheckoutButtonStrategy {
 
         state = this._paymentIntegrationService.getState();
         checkout = state.getCheckoutOrThrow();
+        const sortedShippingOptions: ApplePayJS.ApplePayShippingMethod[] = [];
+        shippingOptions.forEach((option) => {
+            if (option.identifier === selectedOptionId) {
+                sortedShippingOptions.unshift(option);
+            } else {
+                sortedShippingOptions.push(option);
+            }
+        });
```

Comment:
Better to use `.sort` method in this case

Reply:
removed as unnecessary